File: DropZone/DropZone.Web/src/models/jumps/jumpBook.model.ts
import { UserModel } from "../users/user.model";
import { JumpModel } from "./jump.model";

export class JumpBookModel {
	public Id: number;
	public Category: string;
	public ModifiedAt: Date;

	public SportsmanId: number;
	
	public Sportsman: UserModel;

	public Jumps: JumpModel[];
};

File: DropZone/DropZone.Core/Models/JumpBook/JumpBookModel.cs
using System;
using System.Collections.Generic;

namespace DropZone.Core.Models
{
    public class JumpBookModel
    {
        public long Id { get; set; }

        public string Category { get; set; }

        public DateTime ModifiedAt { get; set; }


        public long SportsmanId { get; set; }


        public UserModel Sportsman { get; set; }


        public List<JumpModel> Jumps { get; set; }
    }
}
